distributions/distribution-resources/src/main/resources/conf/services/addons.cfg

```diff
@@ -1,10 +1,12 @@
 # The base installation package of this openHAB instance (default is "standard")
 # Valid options:
 #   - minimal  : Installation only with dashboard, but no UIs or other addons
-#   - standard : Typical installation with all standards UIs
+#   - simple   : Setup for using openHAB purely through UIs - you need to expect MANY constraints in functionality!
+#   - standard : Default setup for normal users, best for textual setup
+#   - expert   : Setup for expoert users, especially for people migrating from openHAB 1.x
```

Comment:
Thanks, this was actually a typo in the original file (fixed it now here https://github.com/openhab/openhab-distro/pull/336)